src/config.ts

```diff
@@ -24,6 +24,9 @@ export type InstallationConfig = rt.Static<typeof installationConfig>;
 
 export function makeInstallationConfig(): rt.Result<InstallationConfig> {
   let pulumiVersion = getInput('pulumi-version');
+  if (pulumiVersion === "undefined") {
```

Comment:
When will this be the string `"undefined"`?

Reply:
This is an awkward side effect of the way we're testing this. When we're setting up the mocked config [here](https://github.com/pulumi/actions/blob/main/src/__tests__/config.test.ts#L27), the "undefined" gets turned into a string. Though maybe we should be using mocks instead for at least the version so we can avoid this :thinking: